lib/client/keystore_test.go

```diff
@@ -102,6 +102,57 @@ func TestKeyStore(t *testing.T) {
 	})
 }
 
+func TestVirtualPath(t *testing.T) {
+	// This test is not Parallel because of env vars.
+	s := newTestAuthority(t)
+	keyStore := NewMemKeyStore()
+
+	// create a test key
+	idx := KeyIndex{"test.proxy.com", "test-user", "root"}
+	key := s.makeSignedKey(t, idx, false)
+
+	// setup virtual path
+	dir := t.TempDir()
+	keyEnv := VirtualPathEnvName(VirtualPathKey, nil)
+	dbEnv := VirtualPathEnvName(VirtualPathDatabase, VirtualPathDatabaseParams(testDBName))
```

Comment:
Can you also cover the other 4 types of virtual paths?

Reply:
Is this comment still relevant, @hugoShaka and @Joerger ?